xapian-core/cluster/cluster.cc

```diff
@@ -0,0 +1,179 @@
+/** @file cluster.cc
+ *  @brief Xapian::Cluster API class
+ */
+
```

Comment:
This should also have a copyright statement; given this seems to take code from the previous cluster branch, it should include the copyright owner from there also.